apps/schema/templates/apidoc.html

```diff
@@ -18,6 +18,6 @@
 </head>
 <body>
     <redoc spec-url="{% static "schema/swagger.yaml" %}"></redoc>
-    <script src="https://cdn.jsdelivr.net/npm/redoc@2.0.0-rc.2/bundles/redoc.standalone.js"> </script>
```

Comment:
Let's keep the version tag in here to make sure our stuff doesn't break over time. I believe the current version is 2.0.0-rc.13